app/assets/stylesheets/homepage.css.scss

```diff
@@ -0,0 +1,6 @@
+.header {
+	width: 100%;
```

Comment:
i thought we deleted these files bootstrap.css and bootstrap.min.css, why are they still in the project